components/record/MetadataField.vue

```diff
@@ -1,52 +1,86 @@
 <template>
   <div
-    v-if="value"
+    v-if="hasValuesForLocale"
```

Comment:
There is something wrong with this check, see for example /en/record/145/URN_RS_NAE_905fa4f7_4345_4d57_bb09_c3910aa3cf06cho. Where the label is shown for "type of object", plus an empty `<ul>`. While there shouldn't be anything shown in my opinion.

Reply:
the logic for this has been fixed, but still needs to be refactored to reduce cognitive complexity.